wafer/locale/ml/LC_MESSAGES/django.po

```diff
@@ -8,205 +8,234 @@ msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2020-11-11 14:41+0000\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: Automatically generated\n"
-"Language-Team: none\n"
+"PO-Revision-Date: 2020-11-17 06:28+0000\n"
+"Last-Translator: Abraham Raji <avronr@tuta.io>\n"
+"Language-Team: Malayalam <https://hosted.weblate.org/projects/wafer/wafer/ml/"
+">\n"
 "Language: ml\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 4.4-dev\n"
 
 #: wafer/compare/admin.py:22
 msgid "Last Modified"
-msgstr ""
+msgstr "അവസാനമായി പരിഷ്‌ക്കരിച്ചതു"
 
 #: wafer/compare/admin.py:27
 msgid "Today"
-msgstr ""
+msgstr "ഇന്ന്"
 
 #: wafer/compare/admin.py:28
 msgid "Since yesterday"
-msgstr ""
+msgstr "ഇന്നലെ മുതൽ"
 
 #: wafer/compare/admin.py:29
 msgid "In the last 7 days"
-msgstr ""
+msgstr "കഴിഞ്ഞ 7 ദിവസത്തിനുള്ളിൽ"
 
 #: wafer/compare/admin.py:30
 msgid "In the last 30 days"
-msgstr ""
+msgstr "കഴിഞ്ഞ 30 ദിവസത്തിനുള്ളിൽ"
 
 #: wafer/compare/admin.py:140
-#, python-format
+#, fuzzy, python-format
```

Comment:
@avronr: Any idea why these are being flagged as fuzzy?

Reply:
Not really but I'll definitely look for something weird.

Reply:
@drnlm said:

> The string is marked as "needs editing" in weblate, which is I think why it's getting marked as fuzzy

You may want to look for those flags in weblate and remove them.

Reply:
So those were translations that were usable but either could be improved or were translations that doesn't really have an exact translation to Malayalam so what I did was simply transliterate. I was hoping to mark them in a way that it would be easy for a future contributor to find.

Reply:
A fuzzy translation won't be used by Django, so I don't think that's the best mechanism to record this.

Comments or glossaries may be better.

Reply:
Got it. Will fix.